src/Domain/Bootstrap.php

```diff
@@ -148,4 +150,12 @@ protected function remove_core_blocks() {
 		remove_action( 'admin_print_footer_scripts', array( 'WC_Block_Library', 'print_script_settings' ), 1 );
 		remove_action( 'init', array( 'WGPB_Block_Library', 'init' ) );
 	}
+
+	/**
+	 * Define the global feature flag
+	 */
+	protected function define_feature_flag() {
+		$woo_options = parse_ini_file( __DIR__ . '/../../blocks.ini' );
+		define( 'WOOCOMMERCE_BLOCKS_PHASE', $woo_options['woocommerce_blocks_phase'] );
```

Comment:
We also should make sure that `$woo_options['woocommerce_blocks_phase']` is a string and maybe sanitized? Should it be checked against a whitelisted set of values?